.gitignore

```diff
@@ -9,3 +9,4 @@ compile_commands.json
 cmake-build-*
 Testing/
 configs/static_config.yaml
+.vscode/
```

Comment:
Same as the other PR: please move next to .idea

Reply:
Done